File: web_lab3.Abstractions/Entities/Book.cs
using System.Collections.Generic;

namespace Abstractions.Entities
{
    public class Book : IEntity<int>
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public List<Sage> Sages { get; set; } = new();

        public List<Order> Orders { get; set; } = new();

        public List<OrdersBooks> OrdersDetails { get; set; } = new();
    }
}

File: Client/src/app/admin/books/book.ts
import {Sage} from '../sages/sage';

export interface Book {
  id?: string;
  name?: string;
  description?: string;
  sages?: Sage[];
}

export interface BookUpdate {
  id?: string;
  name?: string;
  description?: string;
  sages?: number[];
}
